includes/class-wc-stripe-webhook-handler.php

```
@@ -1344,6 +1359,27 @@ public function process_deferred_webhook( $webhook_type, $additional_data, $noti
+		try {
+			$response = WC_Stripe_API::request( [ 'metadata' => $metadata ], 'checkout/sessions/' . $checkout_session_id, 'POST' );
+			if ( ! empty( $response->error->message ) ) {
+				throw new Exception( $response->error->message );
+			}
+		} catch ( Exception $e ) {
+			WC_Stripe_Logger::error( 'Failed to update checkout session metadata: ' . $e->getMessage() );
+
+			// This will be caught by Action Scheduler and logged as an error.
+			throw $e;
+		}
+	}
```

PHPMD (2.15.0)

[error] The method process_checkout_session_metadata is not named in camelCase. (undefined)

(CamelCaseMethodName)